terminatorlib/window.py

```diff
@@ -92,15 +96,44 @@ def __init__(self):
 
             if options.role:
                 self.set_role(options.role)
-            
+
             if options.forcedicon is not None:
                 icon_to_apply = options.forcedicon
 
             if options.geometry:
                 if not self.parse_geometry(options.geometry):
-                    err('Window::__init__: Unable to parse geometry: %s' % 
+                    err('Window::__init__: Unable to parse geometry: %s' %
                             options.geometry)
 
+            if options.guake_key:
```

Comment:
Can we move these command-line options to config file variables?  We can still leave them as command-line options, but merge them into the config file variables with self.config.options_get()